cmd/pj-rehearse/main.go

```diff
@@ -113,6 +95,9 @@ func main() {
 		os.Exit(0)
 	}
 
+	prNumber := jobSpec.Refs.Pulls[0].Number
+	logger = logrus.WithField("PR", prNumber)
```

Comment:
This isn't a field we use upstream

Reply:
Addressed in https://github.com/openshift/ci-operator-prowgen/pull/49/commits/c8b379bfea9953128bf5db1957e6822b5091829e